sourcegraph/jetbrains
# Collapse advanced account settings by default

Fixes https://github.com/sourcegraph/sourcegraph/issues/56572

Previously, the dialog to create a new account with access token showed

* (required) endpoint URL
* (required) access token
* (optional) custom HTTP headers

This PR updates the dialog to collapse advanced and optional settings like "custom HTTP headers" to make it easier for users to create an account with only an endpoint URL and access token.

## Test plan
- `./gradlew :runIde`
- Create new account
- Confirm that custom HTTP headers are collapsed
